DependencyInjection/Compiler/AddVotersPass.php

```diff
@@ -25,13 +25,13 @@ public function process(ContainerBuilder $container)
         $voters = array();
 
         foreach ($container->findTaggedServiceIds('knp_menu.voter') as $id => $tags) {
-            foreach ($tags as $tag) {
-                $priority = isset($tag['priority']) ? (int) $tag['priority'] : 0;
-                $voters[$priority][] = $id;
+            $tag = $tags[0];
```

Comment:
well, for voters, there is no point registering the same voter multiple times (except for hurting performance, but I don't consider this a supported feature, and I doubt anyone wants such feature). The second vote will be the same than the first vote.

And we don't want to forbid having multiple tags. We want to ignore subsequent tags. This is what allows you to override the autoconfigured tag without opting out of all other autoconfiguration: you can add your own tag with a priority, and it will win over the tag added by the autoconfiguration at a priority of 0.
This is consistent with what Symfony does btw. Any tag used to collect services that don't make sense to be collected multiple times is now processing only the first tag. Tags which are used to attach additional info (these often have mandatory attributes) are processing all tags and are not autoconfigured (they cannot be anyway if some attributes are mandatory)